torchvision/ops/deform_conv.py

```diff
@@ -0,0 +1,70 @@
+from typing import Tuple
```

Comment:
Can you import `Tuple` from `from torch.jit.annotations import Tuple`? So that if the user doesn't have `typing` installed, it will still work.